agent-inject/metrics.go

```diff
@@ -21,15 +21,14 @@ var (
 	requestQueue = prometheus.NewGauge(prometheus.GaugeOpts{
 		Namespace: metricsNamespace,
 		Subsystem: metricsSubsystem,
-		Name:      "request_queue_total",
-		Help:      "Total count of webhook requests in the queue",
+		Name:      "request_queue_length",
+		Help:      "Count of webhook requests in the injector's queue",
 	})
 
 	requestProcessingTime = prometheus.NewHistogram(prometheus.HistogramOpts{
 		Namespace: metricsNamespace,
 		Subsystem: metricsSubsystem,
 		Name:      "request_processing_duration_ms",
-		Help:      "Histogram of webhook request processing times in milliseconds",
```

Comment:
I think we should include the help message, just drop the extra "Histogram" bit.